### PR TITLE
initial checkin for basic information 

### PR DESCRIPTION
**Pull request by sgrg1**
just a skeleton code for this requirement. Let me know what else you want me to add.

### CODE REVIEW

File: src/App.scss [hunk 1]
```diff
@@ -7,14 +7,14 @@
 }
 
 .App-header {
```
Comment (martypowell) [inline src/App.scss]: I think all of the css can probably jsut be removed.

File: src/BasicInformation.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { Formik, Field, Form, ErrorMessage } from "formik";
+import * as Yup from "yup";
+
+const BasicInformation = () => {
```
Comment (martypowell) [inline src/BasicInformation.jsx]: I think we want to put this in teh folder structure components/sections or something along those lines. @tmccoy529 thoughts?

File: src/BasicInformation.jsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  return (
+    <Formik
+      initialValues={{
```
Comment (martypowell) [inline src/BasicInformation.jsx]: can we put intiailValues, validationSchema, and onSubmit into their own const above the form please?

File: src/BasicInformation.jsx (hunk 2 above)
Comment (tmccoy529) [reply src/BasicInformation.jsx]: Yeah I think this would be a component inside of the components folder.

File: src/App.scss (hunk 1 above)
Comment (sgrg1) [reply src/App.scss]: removed

File: src/BasicInformation.jsx (hunk 2 above)
Comment (sgrg1) [reply src/BasicInformation.jsx]: done

File: src/BasicInformation.jsx (hunk 3 above)
Comment (sgrg1) [reply src/BasicInformation.jsx]: done. let me know what you think

File: src/components/BasicInformation.jsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        <Field id="address" name="address" type="text" />
+        <ErrorMessage name="address" />
+      </p>
+
+      <button type="submit">Submit</button>
+    </Form>
+  );
+};
+
+export default withFormik({
```
Comment (martypowell) [inline src/components/BasicInformation.jsx]: I was just expecting something like

```js
const validationSchema = () => {
  Yup.object()...
}

const handleSubmit = () => {
}



```

above the form. That will allow us to move the variables into their own files, if this file starts getting too big. Does that make sense?

**Pull request merged**